pymarc/record.py

```diff
@@ -444,6 +444,27 @@ def isbn(self):
             pass
         return None
 
+
+    def lcc_controlnumber(self):
+        """
+        Returns the Library of Congress Classification (LCC)
+        Control Number of the record. 
+        """
+        
+        if self['010']:
+            return self['010'].format_field()
+        return None
+        
+    def lcc_callnumber(self):
+        """
+        Returns the Library of Congress Classification (LCC)
+        Call Number of the record.
+        """
+        
+        if self['050']:
```

Comment:
050 is a repeatable field; should we check for multiples and return one with indicators showing it's assigned by LC< if possible?